CLI/src/CLI.cs

```diff
@@ -267,8 +267,9 @@ private void AddProgramsToRootTable(SymbolTable rootTable, IEnumerable<Program>
         {
             if (_configuration.ExecToStep >= ExecutionStep.SemanticCheck)
             {
-                foreach (var program in programs)
+                foreach (var program in programs.Where(p => !p.IsStacked))
```

Comment:
Following my last remarks, only the main program of a source file need to be in namespace SymbolTable.
Can you also filter nested programs here ?

Reply:
Modifed in order to a stacked program cannot be call by other programs (p.IsMainProgram):
https://github.com/TypeCobolTeam/TypeCobol/pull/1650/commits/3680098497555087b762ae698ab7cf256eb2db6a